keyboards/akko/5087/5087.c

```diff
@@ -0,0 +1,703 @@
+        {0,  26 }, //33
+        {14,  26 }, //34
+        {28,  26 }, //35
+        {42,  26 }, //36
+        {56,  26 }, //37
+        {70,  26 }, //38
+        {84,  26 }, //39
+        {98,  26 }, //40
+        {112,  26 }, //41
+        {126,  26 }, //42
+        {140,  26 }, //43
+        {154,  26 }, //44
+        {168,  26 }, //45
+        {182,  26 }, //46
+        {196,  26 }, //47
+        {210,  26 }, //48
+        {224,  26 }, //49
+        //ROW3
+        {0,  38 }, //50
+        {14,  38 }, //51
+        {28,  38 }, //52
+        {42,  38 }, //53
+        {56,  38 }, //54
+        {70,  38 }, //55
+        {84,  38 }, //56
+        {98,  38 }, //57
+        {112,  38 }, //58
+        {126,  38 }, //59
+        {140,  38 }, //60
+        {154,  38 }, //61
+        {182,  38 }, //62
+        //ROW4
+        {0,  51 }, //63
+        {14,  51 }, //64
+        {28,  51 }, //65
+        {42,  51 }, //66
+        {56,  51 }, //67
+        {70,  51 }, //68
+        {84,  51 }, //69
+        {98,  51 }, //70
+        {112,  51 }, //71
+        {126,  51 }, //72
+        {140,  51 }, //73
+        {182,  51 }, //74
+        {210,  51 }, //75
+        //ROW5
+        {0,  64 }, //76
+        {14,  64 }, //77
+        {28,  64 }, //78
+        {70,  64 }, //79
+        {126,  64 }, //80
+        {140,  64 }, //81
+        {154,  64 }, //82
+        {182,  64 }, //83
+        {196,  64 }, //84
+        {210,  64 }, //85
+        {224,  64 }, //86
+    },
+    {
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,      4,  4,  4,
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,      4,
+		4,  4,  4,  4,  4,  4,  4,  4,  4,  4,  4,          4,      4,
+		4,  4,  4,          4,              4,  4,  4,      4,  4,  4,  4,
+    }
+};
+
+// clang-format on
+bool rgb_matrix_indicators_advanced_kb(uint8_t led_min, uint8_t led_max) {
+    if (!rgb_matrix_indicators_advanced_user(led_min, led_max)) {
+        return false;
+    }
+    if (pwron_flag == false) {
+        map_current_time = timer_read();
+        if (map_current_time < 3000) {
+            rgb_matrix_set_color_all(150, 150, 150);
+        }
+        if (map_current_time >= 3000) {
+            pwron_flag = true;
+            if (!rgb_matrix_get_flags()) {
+                rgb_matrix_set_flags(LED_FLAG_NONE);
+                rgb_matrix_set_color_all(0, 0, 0);
+            }
+        }
+    }
+    return true;
+}
+
+#endif // RGB_MATRIX_ENABLE
+
+void led_test(uint8_t color);
+void clear_eeprom(void);
+
+enum __layers { WIN_B, WIN_W, WIN_FN, MAC_B, MAC_W, MAC_FN };
+enum colors { WHITE, RED, GREEN, BLUE };
+enum colors led_color_status = WHITE;
+
+// clang-format on
+typedef union {
+    uint32_t raw;
+    struct {
+        bool MacMode_flag : 1;
+        bool _WASD_layer_flag : 1;
+    };
+} user_config_t;
+
+user_config_t user_config;
+
```

Comment:
these should be kb config, since it's not user config.

```suggestion
} kb_config_t;

kb_config_t kb_config;

```